docs/resources/http.md.erb

```diff
@@ -40,6 +40,7 @@ where
 * `open_timeout` may be specified for a timeout for opening connections (default to 60)
 * `read_timeout` may be specified for a timeout for reading connections (default to 60)
 * `ssl_verify` may be specified to enable or disable verification of SSL certificates (default to `true`)
+* `max_redirects` may be specified to control how many HTTP Redirects to follow (defaults to `0`)
```

Comment:
I like the single-flag approach, it's concise and powerful, yet obvious how to disable redirects.